proto/api/v1/api.proto

```diff
@@ -436,6 +437,12 @@ service ConnectorService {
     };
   };
 
+  rpc InspectConnector(InspectConnectorRequest) returns (stream opencdc.v1.Record) {
+    option (google.api.http) = {
+      get: "/v1/connectors/{id}/inspect"
```

Comment:
I didn't try this out, but in normal endpoints (no websocket) it is possible to set the option `response_body` and extract a field from the message that's being returned. This could possibly help us make the linter happy and keep the websocket response clean.

So having a message like this:
```
message InspectConnectorResponse {
  opencdc.v1.Record record = 1;
}
```

And add an option here:
```suggestion
      get: "/v1/connectors/{id}/inspect"
      response_body: "record"
```

Again, not sure if this works for websocket endpoints.